pages/linux/ufw.md

```diff
@@ -0,0 +1,24 @@
+## ufw
+
+> Uncomplicated Firewall  
+> frontend for iptables aiming to make configuration of a firewall easier
+
+- Enable ufw
+
+`ufw enable`
+
+- Disable ufw
+
+`ufw disable`
+
+- Add ufw allow rule
+
+`ufw allow {{port}} {{service name}}`
+
+- Add ufw deny rule
+
+`ufw deny {{port}} {{service name}}`
```

Comment:
We earlier agreed to have argument names without spaces, so it should be
{{service_name}}